config/routes.rb

```diff
@@ -1,5 +1,6 @@
 Rails.application.routes.draw do
-  get 'welcome/index'
+  get 'articles/index'
```

Comment:
Cleaner to use `resources :articles, only: [:index, :show]` which you already have defined below